draft-cwbgp-ivy-energy-saving-management.md

```diff
@@ -170,12 +170,21 @@ informative:
    element list node as shown in{{ne-tree}}.
 
 ~~~~
-      augment /ni:network-elements/ni:network-element:
-      +--rw energy-management
-         +--ro energy-monitoring-capability?   boolean
-         +--rw energy-saving-mode?             energy-saving-mode-name
-         +--ro method-name?                    energy-saving-method-name
-         +--ro eer?                            decimal64
+  augment /nw:networks/nw:network/nw:node:
+    +--ro energy-power-consumption
+       +--ro total-energy-consumption?   uint64
+       +--ro saved-energy?               uint64
+       +--ro eer?                        decimal64
```

Comment:
I suggest we also add the following under a node to control it

       +--rw energy-saving-modes
          +--rw energy-saving-mode* [mode]
             +--rw mode       identityref
             +--rw methods
                +--rw method* [method-name]
                   +--rw method-name    identityref